### PR TITLE
MS-4 add text file

### PR DESCRIPTION
**Pull request by ankurbelbase**
@santoshpurbey @raeesaroj @kelsash @SummitStha @AlsaRb

### CODE REVIEW

**Review by AashishGrg — COMMENTED** [review]
I cant understand

File: mysite/Test_MS4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Waffle account created
```
Comment (SummitStha) [inline mysite/Test_MS4.txt:1]: Where is the that account?

**Comment by raeesaroj** [comment]
:+1:

File: mysite/Test_MS4.txt (hunk 1 above)
Comment (ankurbelbase) [reply mysite/Test_MS4.txt:1]: It is in waffle.io

**Comment by AnilKancha** [comment]
👎

**Comment by AashishGrg** [comment]
:1st_place_medal:

**Pull request merged**